SConstruct

```diff
@@ -1368,9 +1357,7 @@ if conf.checkSphinx() and haveInkscape :
 
 else :
 
-	sys.stderr.write( "WARNING : Sphinx not found - not building docs. Check SPHINX build variable.\n" )
-	if not haveInkscape :
-			sys.stderr.write( "WARNING : Inkscape not found - not building documentation graphics. Check INKSCAPE build variable.\n" )
+	sys.stderr.write( "WARNING : Sphinx or Inkscape not found - not building docs. Check SPHINX and INKSCAPE build variables.\n" )
```

Comment:
Can we keep it as separate messages? It's so much more helpful for someone trying to work out whats not working if its informing what's actually missing. We don't want to call `conf.checkSphinx()` twice, so just create a `haveSphinx` variable and use that in the original test, and then here to know when output the sphinx message.